src/Avalonia.Diagnostics/Diagnostics/Convetions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Reflection;
+using Avalonia.Controls;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Diagnostics
+{
+    static class Convetions
+    {
+        public static string DefaultScreenshotsRoot =>
```

Comment:
I don't think we need default folder here. With file picker it's not worth of new apis. With custom handler developer can easily add their own logic instead.